frb_rust/src/ffi/dart_api.rs

```diff
@@ -4,7 +4,16 @@
 
 use std::ffi::c_void;
 
-use dart_sys::{Dart_Handle, Dart_PersistentHandle};
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+pub struct _Dart_Handle {
+    _unused: [u8; 0],
+}
+
+#[allow(non_camel_case_types)]
+pub type Dart_Handle = *mut _Dart_Handle;
+#[allow(non_camel_case_types)]
+pub type Dart_PersistentHandle = Dart_Handle;
```

Comment:
so, is it possible we somehow generate the rust code from c code? (also ok if not)

Reply:
I don't think it will be possible to correctly generate _Dart_Handle